source-code/core2/tsconfig.json

```diff
@@ -5,7 +5,7 @@
 		// DOM because of fetch. TODO: extract fetch from this package and
 		// let fetch be provided by the environment.
 		"lib": ["ESNext", "DOM"],
-		"types": [],
+		"types": ["node"],
```

Comment:
🤔

Reply:
To use the `node:test` runnner. Instead of vitest.

Reply:
Is it also possible to just include `node:test`?

Reply:
Nope :(

The linting rules of don't use "node" are still applying though.